_sass/_site-footer.scss

```diff
@@ -1,46 +1,38 @@
-.site-footer {
-    @extend .page-section;
-    @include clearfix;
-    border-top: 1px solid $colour_borders;
-    .container {
-        @media (min-width: $medium_screen) {
-            padding:  0 2em;
-            position: relative;
-        }
-    }
-}
+$mysoc-footer-breakpoint-sm: $medium_screen;
 
-.tiny-logo {
-    font-size: 1.5em;
-    letter-spacing: -2px;
-    color: $colour_black;
-}
+.dark-theme {
+  // Giving the footer a dark theme is going to require us to override
+  // pretty much all of the default colour variables.
+  $mysoc-footer-background-color: #333;
+  $mysoc-footer-text-color: #acacac;
+  $mysoc-footer-site-name-text-color: #fff;
 
-.footer__about {
-    color: $colour_mid_grey;
-    p {
-        font-size: 0.777777778em;
-        line-height: 1.6em;
-        margin-bottom: 0.5em;
-    }
-    a {
-        color: $colour_dark_grey;
-        text-decoration: underline;
-        &:hover,
-        &:active,
-        &:focus {
-            color:darken($colour_links, 10%);
-        }
-    }
-    nav {
-        position: relative;
-        top: 2.5em;
-        font-size: 0.777777778em;
-        text-align: right;
-        a {
-            margin-right: 1em;
-        }
+  $mysoc-footer-link-text-color: #fff;
+  $mysoc-footer-link-hover-text-color: #fff;
+
+  $mysoc-footer-divider-color: #4b4b4b;
+
+  $mysoc-footer-donate-background-color: #434343;
+  $mysoc-footer-donate-text-color: #fff;
+  $mysoc-footer-donate-button-background-color: $colour_red;
+
+  $mysoc-footer-legal-text-color: #9a9a9a;
+
+  $mysoc-footer-image-path: '../../images/mysoc-footer-dark/';
+
+  @import "ms-footer";
```

Comment:
With all those variables and stuff gone, the only things left in this file will be your manual overrides of some of the footer styles: `.mysoc-footer .column-one-third` and `.mysoc-footer__links`. Tidy!